scripts/jenkins/cloud/ansible/roles/heat-generator/templates/heat-template.yaml

```diff
@@ -91,7 +91,11 @@ resources:
 {%   endif  %}
 {%   if ipv6 and not network.is_conf %}
       ip_version: 6
-{%   else   %}
+{%     if network.is_conf or network.is_mgmt %}
```

Comment:
Can the network.is_conf ever be true here? The parent if on line 92 if only true if it isn't, so it can't be here too right?

Reply:
Good point, it can't :sweat_smile: I'll fix it...

Reply:
Please have another look @matthewoliver !